lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -818,6 +818,18 @@ get_axis_communications_supplemental_authenticity(void *handle,
   return status;
 }
 
+/**
+ * Retrieves the certificate chain from the vendor handle. */
+const char *
+get_axis_communications_certificate_chain(void *handle)
+{
+  if (!handle) return NULL;
+
+  sv_vendor_axis_communications_t *self = (sv_vendor_axis_communications_t *)handle;
+
+  return self->certificate_chain;
```

Comment:
It looks like `certificate_chain` is a `char *` so you should possibly typecast to `const char *` when returning it to avoid some implicit cast warnings